testmon/pytest_testmon.py

```diff
@@ -71,24 +71,33 @@ def pytest_addoption(parser):
         action="store_true",
         dest="no-testmon",
         help="""
-        Turn off (even if activated from config by default). Forced if neither read nor write is possible (debugger 
+        Turn off (even if activated from config by default). Forced if neither read nor write is possible (debugger
         plus test selector)
         """,
     )
 
-
     group.addoption(
         "--testmon-env",
         action="store",
         type=str,
         dest="environment_expression",
         default="",
         help="""
-        This allows you to have separate coverage data within one .testmondata file, e.g. when using the same source 
+        This allows you to have separate coverage data within one .testmondata file, e.g. when using the same source
         code serving different endpoints or django settings.
         """,
     )
 
+    group.addoption(
+        "--testmon-nosort",
+        action="store_true",
```

Comment:
Oh shoot, I didn't mean to change the default behavior. Thanks for catching this, I'll change it to False